pkg/server/plugin/datastore/sql/migration.go

```diff
@@ -185,7 +185,10 @@ func initDB(db *gorm.DB, dbType string, log hclog.Logger) (err error) {
 		return sqlError.Wrap(err)
 	}
 
-	if err := tx.Assign(Migration{Version: latestSchemaVersion}).FirstOrCreate(&Migration{}).Error; err != nil {
+	if err := tx.Assign(Migration{
+		Version:     latestSchemaVersion,
+		CodeVersion: codeVersion.String(),
```

Comment:
It would be great if this could be captured by a test.

Reply:
Yeah, i started down that road and didn't have a clear way to test for that other than reaching into the database manually. I'll add one though.